pages/workoutlist.js

```diff
@@ -40,36 +51,67 @@ export default function WorkoutsList() {
       <StyledHeadline>Choose Your Workout</StyledHeadline>
 
       <WorkoutList>
```

Comment:
I suggest to use semantic HTML elements, whenever possible. Currently, both your `WorkoutList` and `WorkoutCard` are `div` elements. As each workout represents a standalone piece of content, I suggest to use an `article` for your `WorkoutCard`. The `WorkoutList` could be a `section` for grouping all workouts. You could also use an unordered list here (and `li` elements for each workout).